main.js

```diff
@@ -0,0 +1,12 @@
+var days = document.querySelector('.days');
+var h2 = document.querySelector('h2');
+var selectedDay = '';
+
+days.addEventListener('click', function (event) {
+  if (event.target===days){
```

Comment:
This will never be true, you want to check the className of the element that was clicked and make sure it is not the container.